lib/checks/hasUnsupportedNodes.js

```diff
@@ -1,30 +1,29 @@
-const Node = require('../node');
+const {query} = require('math-nodes');
+
 const resolvesToConstant = require('./resolvesToConstant');
 
 function hasUnsupportedNodes(node) {
-  if (Node.Type.isParenthesis(node)) {
```

Comment:
have we totally gotten rid of parenthesis nodes? :o cool

what happens in the case where we want to use them for pedagogical reasons like in grouping like terms?